app/views/dispatch/forecast.erb

```diff
@@ -0,0 +1,65 @@
+<div class="container">
```

Comment:
your formatting is very strange for this file, be sure you're using tabs with 2 spaces! (that's the convention)